plugins/woocommerce/client/blocks/babel.config.js

```diff
@@ -11,14 +11,7 @@ module.exports = {
 		},
 	},
 	presets: [
-		[
-			'@wordpress/babel-preset-default',
-			{
-				targets: {
-					node: 'current',
```

Comment:
I don't know why this was here tbh. I could remove this change, because I don't think we're relying on this config right now, but I also don't think this should be here at all.